analyzers/its/sources/Net6/Net6/Net6.csproj

```diff
@@ -5,7 +5,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>disable</Nullable>
-    <LangVersion>preview</LangVersion>
```

Comment:
It was intended to remove this?

Reply:
Yes. In VS 2022-preview with that line the preview features were enabled at the build was successful, however it changed and the build was failing. Now there is this new way to enable the preview features.

Reply:
As far as I can tell these are 2 very different properties, `LangVersion` is related to the language version, and `EnablePreviewFeatures` is related to APIs. See: https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md#enabling-a-preview-feature

Reply:
In any case, we don't really need `<LangVersion>preview</LangVersion>` now so I can merge it like it is. I was though curious if there was a reason behind the change.